src/runtime_src/core/tools/common/reports/ReportAiePartitions.cpp

```diff
@@ -114,6 +116,7 @@ writeReport(const xrt_core::device* /*_pDevice*/,
       const auto& hw_context = pt_hw_context.second;
 
       const std::vector<std::string> entry_data = {
+        hw_context.get<std::string>("pid"),
```

Comment:
We don't want to print -1 for Ryzen devices. Please do something like:
```
const auto device_class = xrt_core::device_query_default<xrt_core::query::device_class>(dev, xrt_core::query::device_class::type::alveo);
  switch (device_class) {
  case xrt_core::query::device_class::type::alveo:
  {
    hw_context.get<std::string>("pid"),
    break;
  }
```
and print this field only if it is populated

Reply:
Hi @AShivangi , I have already filed a CR to get this info on windows - https://jira.xilinx.com/browse/CR-1204588
If we dont print PID in case of Ryzen devices then users can't provide PID arg and the debug apis behavior will be different on different platforms. So added changes this way.
But that said I am holding off this PR till PID support is added in windows, there is discussion going on this and if it is difficult to provide PID in MCDM flow they will provide some unique identifier in its place.

Reply:
Ok, sounds good!